.gitlab-ci.yml

```
@@ -255,6 +282,57 @@ trigger_devops:
     - if: $CI_COMMIT_BRANCH == "main"
   trigger: lg/identity-devops
 
+review-app:
+  stage: review
+  needs:
+    - job: build-idp-image
+  resource_group: $CI_ENVIRONMENT_SLUG.review-app.identitysandbox.gov
+  image:
+    name: dtzar/helm-kubectl:latest
+  script:
+    - kubectl config get-contexts
+    - export CONTEXT=$(kubectl config get-contexts | grep -v CURRENT | awk '{print $1}' | head -1)
+    - kubectl config use-context "$CONTEXT"
+    - >-
+      helm upgrade --install --namespace review-apps
+      --debug
+      --set idp.image.repository="${ECR_REGISTRY}/identity-idp/review"
+      --set idp.image.tag="${CI_COMMIT_SHA}"
+      --set idp.ingress.enabled=true
```

Adding in worker image and worker image tag as well
```suggestion
      --set worker.image.repository="${ECR_REGISTRY}/identity-idp/review"
      --set worker.image.tag="${CI_COMMIT_SHA}"
      --set idp.ingress.enabled=true
```